config/helper.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// FlexibleBool is a custom boolean type that can be set in multiple ways
```

Comment:
This should include a description what the intended behavior is. "can be set in multiple ways" is too vague. 

Currently, I have to read it to understand what it is.